test/integration/targets/connection_aws_ssm/aws_ssm_integration_test_setup_teardown/files/custom_role.json

```diff
@@ -0,0 +1,37 @@
+{
+    "Version": "2012-10-17",
+    "Statement": [
+        {
+            "Effect": "Allow",
+            "Action": [
+                "cloudwatch:PutMetricData",
+                "ds:CreateComputer",
+                "ds:DescribeDirectories",
+                "ec2:DescribeInstanceStatus",
+                "logs:*",
+                "ssm:*",
```

Comment:
Can you please verify the set of permissions again? We may not need all this permissions.

Reply:
ok, will update the policy document.

Reply:
Done. Updated with new policy